yaksh/templates/yaksh/courses.html

```diff
@@ -140,6 +140,11 @@
                 <a class="test" href="{{URL_ROOT}}/exam/manage/duplicate_course/{{ course.id }}/" data-toggle="tooltip" title="Creates Copy of selected Course as well as its Modules, Lessons/Quizzes" data-placement="top">
                 Clone Course</a>
             </li>
+            <br>
+            <li>
+                <a href="{{URL_ROOT}}/exam/manage/courses/download_course/{{course.id}}" data-toggle="tooltip" title="Download course content in html format" data-placement="top">Download Course
```

Comment:
Should be Download for Offline viewing or something like that! Just Download Course with Download Course csv and Download Yaml course will be pretty confusing!!